firefox-ios/firefox-ios-tests/Tests/XCUITests/DataManagementTests.swift

```diff
@@ -8,11 +8,13 @@ class DataManagementTests: BaseTestCase {
     func cleanAllData() {
         navigator.goto(WebsiteDataSettings)
         mozWaitForElementToExist(app.tables.otherElements["Website Data"])
+        mozWaitForElementToNotExist(app.activityIndicators.firstMatch)
```

Comment:
The spinner is still present sometimes when the test tries to tap "Clear All Website Data".